packages/action-menu/test/action-menu-responsive.test.ts

```diff
@@ -151,6 +157,7 @@ describe('ActionMenu, responsive', () => {
                     },
                 ],
             });
+            await elementUpdated(el);
```

Comment:
Once it's opened and in mobile with a small viewport, why would you `expect(tray).to.be.null;`?

Reply:
I believe that if the test checking forcepopover attribute which renders a popover in mobile instead of the default tray